packages/react-static/src/browser/utils/index.js

```diff
@@ -198,3 +198,23 @@ export function getBasePath() {
     ? ''
     : process.env.REACT_STATIC_BASE_PATH
 }
+
+export function isLocalLink(path) {
+  if (isSSR()) {
+    return false;
+  }
+
+  // script links
+  if (path.indexOf('javascript:') === 0) {
+    return false;
+  }
+
+  try {
+    new URL(path);
+  } catch (e) {
+    // if a path is not parsable by URL its a local relative path
+    return true;
```

Comment:
Do you know if this breaks when using a `config.siteRoot`? When using that, all links are prefixed to be absolute to the site and would technically be a full URL.

Reply:
`getRoutePath` should actually strip that though... So my guess is everything is fine with this, but I just wanted you to confirm ;)